includes/Modules/Idea_Hub.php

```diff
@@ -13,10 +13,13 @@
 use Google\Site_Kit\Core\Modules\Module;
```

Comment:
This import is missing in some other modules too, but this is important for the IDE to know that returning a `Script` from `setup_assets` is a valid return type.
```suggestion
use Google\Site_Kit\Core\Assets\Asset;
use Google\Site_Kit\Core\Modules\Module;
```